adminpages/emailtemplates-edit.php

```diff
@@ -18,7 +18,7 @@
 
 	// Email variables.
 	$email_variables = [
-		'general' => [
+		__( 'General Settings / Membership Info', 'paid-memberships-pro' ) => [

```

Comment:
All of the __() here should be esc_html__()... unless they have HTML in them, then they need to be refactored to escape the translation and keep the required HTML.

Reply:
Of course, this was a .org requirement and we will have more control over our translations and thus more trust of them. If it's a hassle, this can probably wait, but it's a good practice.

Reply:
Is this still the case when the content isn't being echo'd immediately? Currently, we are running this data through esc_html() during output as per "sanitize early, escape late":
`<th><code><?php echo esc_html( $email_variable ); ?></code></th>`

Reply:
It's probably best to escape anyways. Best to be overly cautious